packages/githublight/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@jupyterlab-accessible-themes/jupyterlab-theme-githublight",
+  "version": "0.1.0",
+  "description": "Pitaya Smoothie in JupyterLab: a dark theme inspired by 2018 Pantone of the Year. Designed for low-light condition and colorblindness tested.",
+  "license": "BSD-3-Clause",
+  "author": "jupyter accessibility group",
```

Comment:
This should be either of "Jupyter Development Team" or "Jupyter Accessibility Team"

Reply:
Actually, should be the latter considering this is the one being used on the headers